.github/workflows/pdk-basic.yml

```diff
@@ -0,0 +1,41 @@
+name: PDK basic
+
+on:
+  workflow_call:
+    inputs:
+      container_image:
+        description: Image to use when running tests
+        default: 'puppet/puppet-dev-tools:2022-04-21-e44b72b'
```

Comment:
I would use `puppet/puppet-dev-tools:latest` else we need probably to keep updating the workflow from time to time. People are already able to override this so `stable` as the default seems legit for me

Reply:
latest is actually the oldest looking at the image.  PDK and other tools are outdated.  The latest tag was built for Cd4pe 3.x and has to be maintained to support that older version of cd4pe.